main/src/com/google/refine/importers/SeparatorBasedImporter.java

```diff
@@ -76,6 +76,7 @@ public ObjectNode createParserUIInitializationData(ImportingJob job,
         JSONUtilities.safePut(options, "guessCellValueTypes", false);
         JSONUtilities.safePut(options, "processQuotes", true);
         JSONUtilities.safePut(options, "quoteCharacter", String.valueOf(CSVParser.DEFAULT_QUOTE_CHARACTER));
+        JSONUtilities.safePut(options, "trimStrings", false);
```

Comment:
We could potentially set this to `true` instead - this would enable the option by default in the UI. The importer would still behave in the same way if using an importer configuration generated by a previous version, since you use `false` as default value in `getBoolean` in `readTable`.

I don't have strong feelings about whether this should be enabled by default in the UI. It would perhaps make it more consistent with the other importers which already support trimming strings. And I would expect that workflows where preserving whitespace around strings are quite rare for TSV/CSV. So leaning towards enabling… @ostephens what do you think?

Reply:
Yes, that would be a better option, I think.

Reply:
👍 "enabled", it's a smart default.